### PR TITLE
Update to Laravel 8

### PR DESCRIPTION
**Pull request by nanaya**
Also updated most of the packages except scribe and coding-standard

- [x] Depends on #7751

### CODE REVIEW

**Comment by notbakaneko** [comment]
conflicts 😔

**Comment by nanaya** [comment]
Why is it saying it's out of date with the base branch :thinking:

File: app/Http/Middleware/AuthApi.php [hunk 1]
```diff
@@ -51,10 +48,10 @@ public function handle($request, Closure $next)
...
     {
         $psr = (new PsrHttpFactory(
-            new ServerRequestFactory(),
-            new StreamFactory(),
-            new UploadedFileFactory(),
-            new ResponseFactory()
+            new Psr17Factory(),
+            new Psr17Factory(),
+            new Psr17Factory(),
+            new Psr17Factory(),
```
Comment (notbakaneko) [inline app/Http/Middleware/AuthApi.php:54]: Use the same factory instance? These factories literally have no state

**Review by notbakaneko — CHANGES_REQUESTED** [review]

File: webpack.unmix.js [hunk 2]
```diff
@@ -320,7 +320,7 @@ const resolve = {
     '@images': path.resolve(__dirname, 'public/images'),
     layzr: resolvePath('node_modules/layzr.js/dist/layzr.module.js'),
     ziggy: resolvePath('resources/assets/js/ziggy.js'),
-    'ziggy-route': resolvePath('vendor/tightenco/ziggy/dist'),
+    'ziggy-route': resolvePath('vendor/tightenco/ziggy/dist/index.es.js'),
```
Comment (notbakaneko) [inline webpack.unmix.js:323]: looks like `acorn` 7.2? adds support for `??` but that requires updating to webpack 5 to use it which also needs too many other updates 🤔

**Review by notbakaneko — APPROVED** [review]
seems to work so far 🤔

**Pull request merged**